tests/containers/rootless_podman.pm

```diff
@@ -59,8 +57,21 @@ sub run {
         verify_userid_on_container($runtime, $iname);
     }
     clean_container_host(runtime => $runtime);
+}
+
+sub post_run_hook {
+    my $self = shift;
+    record_info "post-cleanup";
+    $self->SUPER::post_run_hook;
+    $self->select_serial_terminal();
+    assert_script_run "setfacl -x u:$testapi::username /etc/zypp/credentials.d/*" if is_sle;
+}
+
+sub post_fail_hook {
+    my $self = shift;
+    $self->SUPER::post_fail_hook;
     $self->select_serial_terminal();
-    assert_script_run "chmod -R 600 /etc/zypp/credentials.d/*" if is_sle;
+    assert_script_run "setfacl -x u:$testapi::username /etc/zypp/credentials.d/*" if is_sle;
```

Comment:
Not sure how long is the [super chain](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/consoletest.pm#L52) long and our main goal is to clear ACLs, I would prioritize ACL revert, and then we can get logs if there is anything useful for us in this case.

Reply:
Thanks for that man. you forced me to review that approach. i hope it is better now. However i did not prioritize the acl revert, as i think it is important to get the logs before any change.